runtime/manual/basics/modules/publishing_modules.md

```diff
@@ -0,0 +1,89 @@
+---
+Title: Publishing Modules
```

Comment:
Use `oldUrl` here to add a redirect from `/runtime/manual/advanced/publishing/dnt/`

Untested, but something like this:
```
oldUrl:
  - /runtime/manual/advanced/publishing/dnt/
```

Reply:
```suggestion
title: Publishing Modules
```